### PR TITLE
[AnimatedCheckButton] to Nuget.

### PR DESCRIPTION
**Pull request by adlair**

### CODE REVIEW

File: iOS/AnimatedCheckButton/External-Dependency-Info.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
```
Comment (mattleibow) [inline iOS/AnimatedCheckButton/External-Dependency-Info.txt:1]: We shouldn't need this file since I mostly wrote this code.

Comment (mattleibow) [reply iOS/AnimatedCheckButton/External-Dependency-Info.txt:1]: @bholmes Is this assumption right?

**Review by mattleibow — DISMISSED** [review]

File: iOS/AnimatedCheckButton/External-Dependency-Info.txt (hunk 1 above)
Comment (bholmes) [reply iOS/AnimatedCheckButton/External-Dependency-Info.txt:1]: @mattleibow well did you mostly write it or did you write it?  If you actually rewrote the entire library I agree we do not need it.  However if you did draw some inspiration from the code I feel it is fair to give credit by providing attribution.

**Comment by adlair** [comment]
I added new changes on https://github.com/xamarin/XamarinComponents/pull/376.

**Comment by newky2k** [comment]
Replace by #376

**Pull request closed without merging**